docs/working-with-groups.md

```diff
@@ -76,20 +76,22 @@ To retrieve a single Group you need her UUID (for example
 94bbe688-4b1e-4e4e-80e7-e5ba5c4d6db4):
 
 ```java
-OsiamConnector oConnector = [Retrieving an OsiamConnector](create-osiam-connector.md)
-AccessToken accessToken = [Retrieving an AccessToken](login-and-getting-an-access-token.md#retrieving-an-accesstoken)
 Group group = oConnector.getGroup(<GROUP_UUID>, accessToken);
 ```
 (please consider the possible runtimeException which are explained in the
 Javadoc)
 
+It is possible to filter the attributes of the returned resource by specifying the attributes to return as parameters 
+to the method: 
+
+ ```java
+Group group = osiamConnector.getGroup(<GROUP_ID>, accessToken, "displayName", "meta.created");
+ ```
 ## Retrieve all Groups
 
 If you want to retrieve all groups you can call the following method:
 
 ```java
-OsiamConnector oConnector = [Retrieving an OsiamConnector](create-osiam-connector.md)
-AccessToken accessToken = [Retrieving an AccessToken](login-and-getting-an-access-token.md#retrieving-an-accesstoken)
 SCIMSearchResult<Group> searchResult = oConnector.getAllGroups(accessToken);
```

Comment:
`OsiamConnector#getAllGroups` returns `List<Group>` not `SCIMSearchResult<Group>`. Can you change this, too?